docs/architecture.md

```diff
@@ -1,3 +1,82 @@
-# Network Architecture
+# Overview
 
-![Architecture](./assets/architecture.png)
+The Azure Trusted Research Environment (TRE) consists of multiple components all encapsulated in networks with restricted in- & egress traffic. One network for management components and one network per Workspace. All traffic has to explicit allowed by the Application Gateway of the Firewall. Azure TRE is build around the concept of data in a Workspace must not be able to exfiltrate the Workspace.
+
+![Architecture overview](./assets/archtecture-overview.png)
+
+Management consists of two groups of components.
+
+- Management API & Composition Service
+- Shared Services (not implemented yet [#23](https://github.com/microsoft/AzureTRE/issues/23), [#22](https://github.com/microsoft/AzureTRE/issues/21), & [#21](https://github.com/microsoft/AzureTRE/issues/21))
+
+Management API is a services that users can interact with to request changes to Workspaces. To create, update, delete Workspaces or Workspace Services inside each Workspace. The Composition Service is doing the actual work of mutating the state of each Workspace including the Workspace Services.
+
+Ingress/egress components governs all inbound and outbound traffic from the public Internet to and from Azure TRE including the Workspaces. The Firewall Service is managing the egress rules of the Firewall.
+
+Shared Services are services available to all Workspaces. Source Mirror can mirror source repositories such as GitHub, but only allowing read-access, hence data from a Workspace cannot be pushed to a source repository.
+Package Mirror is also a read-only front for developer/researcher application package services like NPM, PyPI, and NuGet and operating system application package services like apt-get and Windows Package Manager (winget).
+
+## Composition Service
+
+The Composition Service is responsible for managing and mutating Workspaces and Workspace Service.
+
+A Workspace is an instance of a Workspace Template. A Workspace Template is implemented as a [Porter](https://porter.sh/) bundle - read more about [Authoring Workspaces Templates](authoring-workspaces.md). A Porter bundle is a full encapsulated bundle with everything needed (binaries, scripts, IoC templates etc.) to provision an instance of Workspace Template.
+The [TRE Administrator](./user-roles.md#tre-administrator) can register a Porter bundle to use the Composition Service to provision instances of the Workspace Templates.
+To do so requires:
+
+1. The Porter bundle to be pushed to the Azure Container Registry (ACR).
+1. Registering the Workspace through the Management API.
+
+### Provision a Workspace
+
+![Composition Service](./assets/composition-service.png)
```

Comment:
Same as for the diagram above - We should move text out of the diagram for readability.

Reply:
Removed the text and enlarged the number reference icons